docs/conf.py

```diff
@@ -251,11 +251,17 @@
 # How to display URL addresses: 'footnote', 'no', or 'inline'.
 #texinfo_show_urls = 'footnote'
 
-extlinks = {'redmine': ('https://pulp.plan.io/issues/%s', '#'),
-            'fixedbugs': ('https://pulp.plan.io/projects/pulp/issues?c%%5B%%5D=tracker&c%%5B%%5D='
-                          'status&c%%5B%%5D=priority&c%%5B%%5D=cf_5&c%%5B%%5D=subject&c%%5B%%5D=a'
-                          'uthor&c%%5B%%5D=assigned_to&c%%5B%%5D=cf_3&f%%5B%%5D=cf_4&f%%5B%%5D=tr'
-                          'acker_id&f%%5B%%5D=&group_by=&op%%5Bcf_4%%5D=%%3D&op%%5Btracker_id%%5D'
-                          '=%%3D&set_filter=1&sort=priority%%3Adesc%%2Ccf_5%%3Adesc%%2Cid%%3Adesc'
-                          '&utf8=%%E2%%9C%%93&v%%5Bcf_4%%5D%%5B%%5D=%s&v%%5Btracker_id%%5D%%5B%%5'
-                          'D=1', 'bugs fixed in ')}
+_bugs_link = ('https://pulp.plan.io/projects/%s/issues?c%%%%5B%%%%5D=tracker&c%%%%5B%%%%5D='
+              'status&c%%%%5B%%%%5D=priority&c%%%%5B%%%%5D=cf_5&c%%%%5B%%%%5D=subject&c%%%%'
+              '5B%%%%5D=author&c%%%%5B%%%%5D=assigned_to&c%%%%5B%%%%5D=cf_3&f%%%%5B%%%%5D=c'
+              'f_4&f%%%%5B%%%%5D=tracker_id&f%%%%5B%%%%5D=&group_by=&op%%%%5Bcf_4%%%%5D=%%%'
+              '%3D&op%%%%5Btracker_id%%%%5D=%%%%3D&set_filter=1&sort=priority%%%%3Adesc%%%%'
+              '2Ccf_5%%%%3Adesc%%%%2Cid%%%%3Adesc&utf8=%%%%E2%%%%9C%%%%93&v%%%%5Bcf_4%%%%5D'
+              '%%%%5B%%%%5D=%s&v%%%%5Btracker_id%%%%5D%%%%5B%%%%5D=1')
+
+_projects = ['pulp', 'pulp_rpm', 'pulp_puppet', 'pulp_python', 'pulp_ostree', 'pulp_docker']
+
+# create directive fixedbugs_<project name> for each project
+extlinks = {'fixedbugs_' + project: (_bugs_link % (project, '%s'), 'bugs fixed in ')
```

Comment:
Even similar string urls now will be dis-similar when the plugins are versioned independantly with pulp3. So even if the strings are the same I think writing them out explicitly would be easy and good.